collections/ancestrynew.js

```diff
@@ -222,60 +197,62 @@ function parseAncestryNew(htmlstring, familymembers, relation) {
+    }
+    for(var i = 0; i < personFacts.ResearchFamily.HalfSiblings.length; i++) {
+        var halfsibling = personFacts.ResearchFamily.HalfSiblings[i];
+        if (familymembers && exists(halfsibling.ClickUrl)) {
+            await getAncestryNewTreeFamily(famid++, halfsibling.Id, halfsibling.FullName.trim(), "halfsibling", halfsibling.ClickUrl);
+        }
+    }
+    for(var i = 0; i < personFacts.ResearchFamily.Siblings.length; i++) {
+        var sibling = personFacts.ResearchFamily.Siblings[i];
+        if (familymembers && exists(sibling.ClickUrl)) {
+            await getAncestryNewTreeFamily(famid++, sibling.Id, sibling.FullName.trim(), "sibling", sibling.ClickUrl);
+        }
+    }
+    for(var i = 0; i < personFacts.ResearchFamily.Fathers.length; i++) {
+        var father = personFacts.ResearchFamily.Fathers[i];
+        if (familymembers) {
+            if (exists(father.ClickUrl)) {
+                await getAncestryNewTreeFamily(famid++, father.Id, father.FullName.trim(), "father", father.ClickUrl);
+            }
+        } else if (exists(relation.title)) {
+            if (isChild(relation.tite)) {
+                if (personCard.personId !== father.Id) {
+                    childlist[relation.proid] = $.inArray(father.Id, unionurls);
+                    profiledata["parent_id"] = $.inArray(father.Id, unionurls);
                 }
             }
-
-            if (exists(url)) {
-                var itemid = parseAncestryNewId(url);
-                if (familymembers) {
-                    var name = $(person[x]).find(".userCardTitle").text();
-                    getAncestryNewTreeFamily(famid, itemid, name, title, url);
-                    famid++;
-                } else if (exists(relation.title)) {
-                    if (isChild(relation.title)) {
-                        if (isParent(title)) {
-                            if (focusURLid !== itemid) {
-                                childlist[relation.proid] = $.inArray(itemid, unionurls);
-                                profiledata["parent_id"] = $.inArray(itemid, unionurls);
-                            }
-                        }
-                    } else if (isSibling(relation.title)) {
-                        if (isParent(title)) {
-                            siblingparents.push(itemid);
-                        }
-                    }
+        }
+    }
+    for(var i = 0; i < personFacts.ResearchFamily.Mothers.length; i++) {
+        var mother = personFacts.ResearchFamily.Mothers[i];
+        if (familymembers) {
+            if (exists(mother.ClickUrl)) {
+                await getAncestryNewTreeFamily(famid++, mother.Id, mother.FullName.trim(), "mother", mother.ClickUrl);
+            }
+        } else if (exists(relation.title)) {
+            if (isChild(relation.tite)) {
+                if (personCard.personId !== mother.Id) {
+                    childlist[relation.proid] = $.inArray(mother.Id, unionurls);
```

Comment:
I know this is nitpicking, but @mamsterla @eljeffeg perhaps we should move away from JQuery to standard JS libraries - e.g. `unionurls.includes(mother.id)`?

This is a far bigger task to do than this, but I'm just suggesting

Reply:
No objection if someone wants to spend the time to convert it.